sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj

```diff
@@ -22,6 +22,8 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+    <!--STJ reference is temporary until Azure.Core releases with updated version-->
+    <PackageReference Include="System.Text.Json" />
```

Comment:
The code in this library wasn't broken yet but would have broken once Azure.Core released with this package upgrade. This can be removed after Azure.Core releases.

Reply:
Do we have an issue tracking this?

Reply:
We do now - https://github.com/Azure/azure-sdk-for-net/issues/45660 ☺️